core/src/test/java/org/frankframework/pipes/PasswordGeneratorPipeTest.java

```diff
@@ -43,9 +35,10 @@ public void testDoPipe() throws Exception {
         pipe.setNumOfLCharacters(4);
         pipe.setNumOfUCharacters(2);
         pipe.setNumOfSigns(2);
-        pipe.setUseSecureRandom(false);
         pipe.configure();
+
         PipeRunResult res = doPipe(pipe, "pipey", session);
-        assertEquals(11, res.getResult().asString().length());
+        assertEquals(11, res.getResult().size());
```

Comment:
De `size()` is niet altijd gegarandeerd, als je de `.asString()` kan doen is dat zekerder denk ik.

Reply:
Oke, zet ik terug. Unittest slaagde uiteraard wel gewoon, dus ik dacht dat het dan wel prima was in deze use-case.

Reply:
Ik zeg het hier puur omdat de size-berekening van Message misschien veranderd, of een andere implementatie misschien veranderd, waardoor de test kan falen zonder dat het `.asString()` resultaat veranderd.

Reply:
Oke helder, dank je.